draft/spec/index.html

```diff
@@ -868,6 +868,34 @@ <h2>Extensions</h2>
             </p>
         </blockquote>
     </section>
+    
+    <section id="filesystem-features">
+        <h2>Filesystem features</h2>
+        <p>
+            In order to maximise the compatibility of the OCFL with different filesystems, and thus improve the 
+            portability of OCFL Objects betwwen different systems, some restrictions on the use of certain 
+            filesystem features are necessary. If the preservation of non-OCFL-compliant features is required then
+            data MUST be wrappered in a suitable disk or filesystem image format which OCFL can treat as a regular
```

Comment:
Add (e.g., ISO or Tar)?

Reply:
wrappered or simply wrapped? Or would encapsulated be better?

If we say "tar" I think all lowercase is conventional (see, e.g. https://en.wikipedia.org/wiki/Tar_(computing) ). I'm +0 on adding these examples, it feels a little awkward to add things like that without good links to specs etc. and without consideration of what bias we might be inserting into choice of format.